src/ui/modules/blog/BlogList/FilterList.tsx

```
@@ -0,0 +1,31 @@
+  if (!categories) return null;
+
+  return (
+    <fieldset>
+      <legend className="sr-only">Filter by category</legend>
+
+      <div className="flex flex-wrap justify-start gap-1 max-sm:justify-center">
+        <Suspense>
+          <Filter label="All" />
+
+          {categories?.map((category, key) => (
+            <Filter label={category.title} value={category.slug?.current} key={key} />
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)